### ISSUE TITLE
Establish environment

### ISSUE THREAD

**Issue by Mofr77** [Junyi42/sd-dino#15]
Hello, I am very interested in your work, but I encountered some difficulties when setting up the environment. I followed the steps in the README, but there seems to be some problem somewhere, and I don't know how to fix it.
![image](https://github.com/Junyi42/sd-dino/assets/154524452/ef04fe75-8d1a-40e6-99f0-bb9b9d52deea)

**Comment by Junyi42** [comment]
Hi @Mofr77 ,

Thanks so much for your interest in our work!

Please refer to the following issues: #11, https://github.com/NVlabs/ODISE/issues/32#issuecomment-1722608838. Feel free to let me know if it works, thanks!

Best,
Junyi

**Comment by Mofr77** [comment]
> Hi @Mofr77 ,
> 
> Thanks so much for your interest in our work!
> 
> Please refer to the following issues: #11, [NVlabs/ODISE#32 (comment)](https://github.com/NVlabs/ODISE/issues/32#issuecomment-1722608838). Feel free to let me know if it works, thanks!
> 
> Best, Junyi

Hi! Thanks for your solutions,My undergraduate graduation project was inspired by you. I am studying your thesis. Although this method you gave to me does not work for me, I have solved it. I re-established an environment and ran pip install -e . on mask2former and detectron2 project by project. Updated the version of pytorch cuda and found that it can be used. But I also used the latest version of pytorch-cuda yesterday, and it didn’t work, which is very strange lol
